nixpkgs-review on this kicked out the same error as the ofborg runs, it looks like:

```console
$ nix log /nix/store/iqhwjihff0br43xlw1q5knd57l9z88ck-setuptools-44.0.0-sdist.tar.gz.drv
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/vp039y542nr4qq62lxm7d3sd4piix1fi-setuptools-44.0.0-source
source root is setuptools-44.0.0-source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
applying patch /nix/store/0qzs4kq2lvl09k80nf54bl00hd2ylfam-tag-date.patch
patching file setup.cfg
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
adding minimal entry_points
Regenerating egg_info
Traceback (most recent call last):
  File "setup.py", line 9, in <module>
    import setuptools
  File "/private/tmp/nix-build-setuptools-44.0.0-sdist.tar.gz.drv-0/setuptools-44.0.0-source/setuptools/__init__.py", line 18, in <module>
    import setuptools.version
  File "/private/tmp/nix-build-setuptools-44.0.0-sdist.tar.gz.drv-0/setuptools-44.0.0-source/setuptools/version.py", line 1, in <module>
    import pkg_resources
  File "/private/tmp/nix-build-setuptools-44.0.0-sdist.tar.gz.drv-0/setuptools-44.0.0-source/pkg_resources/__init__.py", line 36, in <module>
    import email.parser
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/email/parser.py", line 12, in <module>
    from email.feedparser import FeedParser
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/email/feedparser.py", line 27, in <module>
    from email import message
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/email/message.py", line 16, in <module>
    import email.charset
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/email/charset.py", line 13, in <module>
    import email.base64mime
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/email/base64mime.py", line 40, in <module>
    from email.utils import fix_eols
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/email/utils.py", line 29, in <module>
    import urllib
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/urllib.py", line 1448, in <module>
    from _scproxy import _get_proxy_settings, _get_proxies
ImportError: No module named _scproxy
Traceback (most recent call last):
  File "bootstrap.py", line 64, in <module>
    __name__ == '__main__' and main()
  File "bootstrap.py", line 61, in main
    run_egg_info()
  File "bootstrap.py", line 54, in run_egg_info
    subprocess.check_call(cmd)
  File "/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/lib/python2.7/subprocess.py", line 190, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/nix/store/brf170js3qrvv52pmka7igk79jhw276c-python-2.7.18.5/bin/python2.7', u'setup.py', u'egg_info']' returned non-zero exit status 1
```

--- a/pkgs/development/interpreters/python/cpython/2.7/default.nix
+++ b/pkgs/development/interpreters/python/cpython/2.7/default.nix
@@ -31,6 +31,10 @@
 , rebuildBytecode ? true
 , reproducibleBuild ? false
 , enableOptimizations ? false
+, strip2to3 ? false
+, stripConfig ? false
+, stripIdlelib ? false
+, stripTests ? false
 , pythonAttr ? "python${sourceVersion.major}${sourceVersion.minor}"
 }:
 
@@ -318,6 +322,16 @@ in with passthru; stdenv.mkDerivation ({
     postFixup = ''
       # Include a sitecustomize.py file. Note it causes an error when it's in postInstall with 2.7.
       cp ${../../sitecustomize.py} $out/${sitePackages}/sitecustomize.py
+    '' + optionalString strip2to3 ''
+      rm -R $out/bin/2to3 $out/lib/python*/lib2to3
+    '' + optionalString stripConfig ''
+      rm -R $out/bin/python*-config $out/lib/python*/config-*
+    '' + optionalString stripIdlelib ''
+      # Strip IDLE
+      rm -R $out/bin/idle* $out/lib/python*/idlelib
+    '' + optionalString stripTests ''
+      # Strip tests
+      rm -R $out/lib/python*/test $out/lib/python*/**/test{,s}
     '';
 
     enableParallelBuilding = true;

--- a/pkgs/development/misc/resholve/default.nix
+++ b/pkgs/development/misc/resholve/default.nix
@@ -1,19 +1,37 @@
 { lib
-, stdenv
+, pkgs
 , pkgsBuildHost
 , ...
 }:
 
 let
-  pkgs = import ../../../.. {
-    inherit (stdenv.hostPlatform) system;
-    # Allow python27 with known security issues only for resholve,
-    # see issue #201859 for the reasoning
-    # In resholve case this should not be a security issue,
-    # since it will only be used during build, not runtime
-    config.permittedInsecurePackages = [ pkgsBuildHost.python27.name ];
+  python27' = (pkgsBuildHost.python27.overrideAttrs (old:
+    {
+      # Overriding `meta.knownVulnerabilities` here, see #201859 for why it exists
+      # In resholve case this should not be a security issue,
+      # since it will only be used during build, not runtime
+      meta = (old.meta or { }) // { knownVulnerabilities = [ ]; };
+    }
+  )).override {
+    self = python27';
+    pkgsBuildHost = pkgsBuildHost // { python27 = python27'; };
+    # strip down that python version as much as possible
+    openssl = null;
+    bzip2 = null;
+    readline = null;
+    ncurses = null;
+    gdbm = null;
+    sqlite = null;
+    libffi = null;
+    rebuildBytecode = false;
+    stripBytecode = true;
+    strip2to3 = true;
+    stripConfig = true;
+    stripIdlelib = true;
+    stripTests = true;
+    enableOptimizations = false;
   };
-  callPackage = lib.callPackageWith pkgs;
+  callPackage = lib.callPackageWith (pkgs // { python27 = python27'; });
   source = callPackage ./source.nix { };
   deps = callPackage ./deps.nix { };
 in

